src/ai-did-you-mean-this/azext_ai_did_you_mean_this/custom.py

```diff
@@ -32,7 +32,7 @@
 # Commands
 # note: at least one command is required in order for the CLI to load the extension.
 def show_extension_version():
-    print(f'Current version: 0.1.1')
+    print(f'Current version: 0.2.0')
```

Comment:
If there is no placeholder, then there is no need to use f-string.
```suggestion
    print('Current version: 0.2.0')
```

Reply:
Removed use of f-string.